lib/mirage_impl_arpv4.ml

```diff
@@ -13,7 +13,7 @@ let arp_conf = object
   method name = "arp"
   method module_name = "Arp.Make"
   method! packages =
-    Key.pure [ package ~min:"1.0.0" ~max:"2.0.0" "arp-mirage" ]
+    Key.pure [ package ~min:"2.0.0" ~max:"3.0.0" "arp-mirage" ]
```

Comment:
these don't follow the pattern of `y` being bumped in `x.y.z` translating to an interface break?